internal/wrappers/byor-http.go

```diff
@@ -50,7 +50,12 @@ func (b *ByorHTTPWrapper) Import(projectID, uploadURL string) (string, error) {
 	case http.StatusUnauthorized:
 		return "", getError(decoder, errorConstants.StatusUnauthorized)
 	case http.StatusInternalServerError:
-		return "", getError(decoder, errorConstants.StatusInternalServerError)
+		errorModel := ErrorModel{}
+		decodeErr := decoder.Decode(&errorModel)
+		if decodeErr != nil {
+			return "", errors.Errorf("Parsing error model failed - %s", err.Error())
```

Comment:
return the const of error in byor but without code and message

Reply:
fixed

Reply:
seems like not fixed yet?